misc/ls-extensions/modules/bal-shell-service/build.gradle

```diff
@@ -69,4 +69,3 @@ jar {
     }
 }
 
-task wrapper(){}
```

Comment:
I am a bit confused, why the wrapper task is overwritten to do nothing here. With that, you can't update the wrapper via the CLI (`./gradlew wrapper --gradle-version 8.7`). @HirumalPriyashan Do you remember why you added this line?